config/locales/de.yml

```diff
@@ -136,6 +146,18 @@ de:
       away: "Team 2"
       score: "Ergebnis"
       points: "Punkte"
+    metric:
+      elo: "ELO"
+      win_loss: "Gewinne-Niederlagen"
+      true_skill: "Wahres Können"
+    gamemode:
+      ko: "KO"
+      ko_group: "Gruppen-KO"
+      double_elimination: "Doppel-KO-System"
+      round_robin: "Runder Robin"
+      two_halfs: "Zwei Halbe"
```

Comment:
The correct plural of 'half' is 'halves'. The correct translation was sadly already defined in #19. Therefore it should be "Hin- und Rückrunde".